tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -243,7 +248,9 @@ describe('Lobby - P1 Perspective', () => {
     cy.readyOpponent();
     assertGameStarted();
   });
-  it.skip('[Feature Missing] Loads lobby after page refresh', () => {
-    expect(true).to.eq(false, 'Empty Test');
+  // TODO: Rewrite this test when auth is supported in the lobby
+  it.skip('Redirects to game list after page refresh', () => {
+    cy.reload();
+    cy.hash().should('equal', '#/');
```

Comment:
Same as above -- could confirm that you can join another game and have it work correctly